compose/mpp/demo/src/wasmJsMain/kotlin/androidx/compose/mpp/demo/main.js.kt

```diff
@@ -19,14 +19,11 @@ package androidx.compose.mpp.demo
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.remember
 import androidx.compose.ui.ExperimentalComposeUiApi
-import androidx.compose.ui.window.CanvasBasedWindow
+import androidx.compose.ui.window.ComposeViewport
 
 @OptIn(ExperimentalComposeUiApi::class)
 fun main() {
-    CanvasBasedWindow("Compose/JS sample", canvasElementId = "canvas1") {
```

Comment:
Here I see some lines deleted, but no new API used here :) 
So fun main does nothing here. Not intentional I guess?

Reply:
yep, sorry, this was due to commit not being actually happened because of signup configuration